composer/trainer/devices/device_tpu.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""The TPU device used for training."""
+
+from __future__ import annotations
+
+import logging
+from contextlib import contextmanager
+from typing import Generator, TypeVar, Union
+
+import torch
+
+from composer.core.precision import Precision
+from composer.trainer.devices.device import Device, T_nnModule
+import torch_xla
```

Comment:
Can this be a conditional dependency? See https://github.com/mosaicml/composer/blob/dev/STYLE_GUIDE.md#61-external-dependencies